.github/workflows/run_tests.yaml

```diff
@@ -36,4 +36,14 @@ jobs:
       # upstream so it doesn't have to rely on order in case of equality.
       # Until then this test is disabled but similar tests are internalized in
       # sklearn_numba_dpex.
+      
+      # TODO: test_kmeans_relocated_clusters on the sklearn side relies on the 
+      # arbitrary tie-breaking behavior of the np.argpartition function. The 
+      # within-partition order should be considered an implementation detail and 
+      # sklearn_numba_depx own kernels for argpartition do not attempt to replicate the 
```

Comment:
The suggestion feature does not show-up on this line but there is a typo:

`sklearn_numba_depx` => `sklearn_numba_dpex`